<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IcecastFilterWidget</class>
 <widget class="QWidget" name="IcecastFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="LineEdit" name="filter">
     <property name="hint" stdset="0">
      <string>Search Icecast stations</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="options">
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_sort_genre_alphabetically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort by genre (alphabetically)</string>
   </property>
  </action>
  <action name="action_sort_genre_popularity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort by genre (by popularity)</string>
   </property>
  </action>
  <action name="action_sort_station">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sort by station name</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
